cellprofiler/modules/threshold.py

```diff
@@ -106,99 +102,87 @@ def create_settings(self):
-                        probability map output of the <b>ClassifyPixels</b> module (in which case, a value of
-                        0.5 should be chosen). If the input image is already binary (i.e., where the foreground
-                        is 1 and the background is 0), a manual value of 0.5 will identify the objects.</dd>
-                    </dl>
-                </li>
-                <li>
-                    <i>{TM_MEASUREMENT}:</i> Use a prior image measurement as the threshold. The measurement
-                    should have values between zero and one. This strategy can be used to apply a
-                    pre-calculated threshold imported as per-image metadata via the <b>Metadata</b> module.
-                    <dl>
-                        <dd><img src="memory:{PROTIP_RECOMEND_ICON}">&nbsp; Like manual thresholding, this
-                        approach can be useful when you are certain what the cutoff should be. The difference
-                        in this case is that the desired threshold does vary from image to image in the
-                        experiment but can be measured using another module, such as one of the <b>Measure</b>
-                        modules, <b>Threshold</b> or an <b>Identify</b> module.</dd>
-                    </dl>
-                </li>
-            </ul>
-            <p><b>References</b></p>
-            <ul>
-                <li>Sezgin M, Sankur B (2004) "Survey over image thresholding techniques and quantitative
-                performance evaluation." <i>Journal of Electronic Imaging</i>, 13(1), 146-165. (<a href=
-                "http://dx.doi.org/10.1117/1.1631315">link</a>)
-                </li>
-            </ul>
-            <p></p>
+            doc="""\
+*(Used only if {TS_GLOBAL} is selected for thresholding scope)*
+
+The intensity threshold affects the decision of whether each pixel
+will be considered foreground (region(s) of interest) or background. A
+higher threshold value will result in only the brightest regions being
+identified, whereas a lower threshold value will include dim regions.
+You can have the threshold automatically calculated from a choice of
+several methods, or you can enter a number manually between 0 and 1
+for the threshold.
+
+Both the automatic and manual options have advantages and disadvantages.
+
+|image0|  An automatically-calculated threshold adapts to changes in
+lighting/staining conditions between images and is usually more
+robust/accurate. In the vast majority of cases, an automatic method is
+sufficient to achieve the desired thresholding, once the proper method
+is selected. In contrast, an advantage of a manually-entered number is
+that it treats every image identically, so use this option when you have
+a good sense for what the threshold should be across all images. To help
+determine the choice of threshold manually, you can inspect the pixel
+intensities in an image of your choice. {HELP_ON_PIXEL_INTENSITIES}.
```

Comment:
Remove the `.` at the end of this line. `HELP_ON_PIXEL_INTENSITIES` ends with a `.`